src/Microsoft.PowerShell.Commands.Management/Microsoft.PowerShell.Commands.Management.csproj

```diff
@@ -47,7 +47,7 @@
 
   <ItemGroup>
     <!-- the following package(s) are from https://github.com/dotnet/corefx -->
-    <PackageReference Include="System.ServiceProcess.ServiceController" Version="5.0.0-rc.1.20451.14" />
+    <PackageReference Include="System.ServiceProcess.ServiceController" Version="5.0.0-rc.2.20479.6" />
```

Comment:
The officially released package versions for RC.2 is `5.0.0-rc.2.20475.5`, see https://www.nuget.org/packages/System.ServiceProcess.ServiceController/5.0.0-rc.2.20475.5

Reply:
just a question. Is the plan to have an RC of PowerShell 7.1 based on RTW/RTM .NET 5.0? That is, a release candidate for PowerShell that is built on top of .NET 5.0 and not an RC of 5??